shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -398,14 +398,19 @@ - (void)dispatchMouseEvent:(NSEvent*)event phase:(FlutterPointerPhase)phase {
 }
 
 - (void)dispatchKeyEvent:(NSEvent*)event ofType:(NSString*)type {
-  [_keyEventChannel sendMessage:@{
+  NSMutableDictionary* keyMessage = [[NSMutableDictionary alloc] init];
+  [keyMessage addEntriesFromDictionary:@{
```

Comment:
Optional nit: the code might be slightly simpler (and maybe more efficient, but that won't matter in practice) if you just initialize `keyMessage` to `[@{...} mutableCopy]`.

Reply:
SG. Done